docs/tutorial/08_deploying-to-localhost.md

```diff
@@ -0,0 +1,122 @@
+# 8. Deploying to localhost
+
+To test the contracts it's enough to start a Buidler EVM, run the test, and then stop it. But now that we are communicating with a user interface we need to simulate 
+the real blockchain more faithfully by leaving the EVM running. This means we need to deploy the contract we're developing for with a script, rather than as part of a test.
+
+Create a new directory inside the project's root directory, `scripts`, and paste this file as `scripts/deploy.js`:
+
+```js
+const contractName = "Token"
+
+// This is a script for deploying contracts. You can adapt it to deploy yours.
+const main = async () => {
```

Comment:
I think the sooner people get used to `const <name> = <lambda>` the better, it's just confusing to us because we grew on C like languages where that wasn't the case, but I'll change it.